custom-dashboards/.applier/group_vars/all.yml

```diff
@@ -1,83 +1,146 @@
+dashboard_namespace: openshift-monitoring
```

Comment:
@ramius345 let's default this to a separate namespace called `custom-dashboards`. Otherwise when we run cleanup, we might accidentally delete openshift-monitoring.

Reply:
Got it, ill fix it.

Reply:
Done.